routers/private/hook.go

```diff
@@ -428,14 +429,14 @@ func HookPostReceive(ctx *macaron.Context, opts private.HookOptions) {
 					branch = fmt.Sprintf("%s:%s", repo.OwnerName, branch)
 				}
 				results = append(results, private.HookPostReceiveBranchResult{
-					Message: true,
+					Message: setting.Git.PullRequestPushMessage && repo.AllowsPulls(),
```

Comment:
I think that this would have been good enough, though.
```golang
Message: repo.AllowsPulls(),
```

Reply:
I more or less agree, but we may as well make it configurable for those who don't want the extra messages.